curvefs/src/metaserver/partition_cleaner.cpp

```diff
@@ -28,18 +28,6 @@ namespace curvefs {
 namespace metaserver {
 
 bool PartitionCleaner::ScanPartition() {
-    if (partition_->GetInodeNum() == 0) {
-        LOG(INFO) << "Inode num is 0, delete partition from metastore";
-        MetaStatusCode ret = DeletePartition();
-        if (ret == MetaStatusCode::OK) {
-            return true;
-        } else {
-            LOG(WARNING) << "delete partition from copyset fail, ret = "
-                         << MetaStatusCode_Name(ret);
-            return false;
-        }
-    }
-
```

Comment:
you should put below if condition into for-loop, because the leader may be transferred.

Reply:
the new leader will cover this situation